### DISCUSSION TITLE
Can I use single and paired end fastq data from same species to generate single transcriptomic assembly in rnaSPADES?

### DISCUSSION THREAD

**Discussion by ReemaSingh** [ablab/spades#1172] [Q&A] [answered]
Dear Spades community.

I am using rnaSpades to generate transcriptomics assembly from my species of interest. I have a total of 45 RNAseq datasets - 42 (of 45) are single-end FASTQ files whereas 3 (of 45) are paired-end FASTQ files. I would like to use all 45 FASTQ files to generate a single assembly. Therefore, I would like to know if is it acceptable (as well as correct) to use both paired and single-end FASTQ files in a single rnaSPADES command. Please see below the example:

rnaspades.py --pe1-1 File1_1.fq.gz  --pe1-2 File1_2.fq.gz  --pe1-1 File2_1.fq.gz  --pe1-2 File2_2.fq.gz --pe1-1 File3_1.fq.gz  --pe1-2 File3_2.fq.gz  --s1 File1.fq.gz --s1 File2.fq.gz --s1 FileN.fq.gz --threads 20 -o OUTPUT

I would highly appreciate any suggestions on this. 

Thank you,
Reema,

**Comment by asl** [comment, marked as answer]
Yes, you can provide unlimited number of input files to rnaSPAdes

**Reply by ReemaSingh** [reply]
Thank you @asl. I appreciate your response.
